apis/kubekey/v1alpha1/cluster_types.go

```diff
@@ -177,6 +177,7 @@ type RegistryConfig struct {
 	RegistryMirrors    []string `yaml:"registryMirrors" json:"registryMirrors,omitempty"`
 	InsecureRegistries []string `yaml:"insecureRegistries" json:"insecureRegistries,omitempty"`
 	PrivateRegistry    string   `yaml:"privateRegistry" json:"privateRegistry,omitempty"`
+	NamespaceOverride  string   `yaml:"namespaceOverride" json:"namespaceOverride,omitempty"`
```

Comment:
The `v1alpha1` version of the struct does not require this field. You can remove it.

Reply:
done